fastlane/Fastfile

```diff
@@ -0,0 +1,121 @@
+fastlane_version "2.63.0"
+
+skip_docs
+
+desc "Run the tests"
+lane :test do
+  scan(scheme: "JOSESwift")
+end
+
+desc "Lint Swift files"
+lane :lint do
+  # Todo: Don't ignore exit status (#98)
+  swiftlint(ignore_exit_status: true)
```

Comment:
Are we ok with ignoring the `swiftlint` exit status for now?

Reply:
We have to, for now. Otherwise, we can't get past that step in the automation because we get a whole bunch of warnings errors currently. We have to get rid of them first and make a decision which ones to ignore. I setup #98 for that.